docker/freertos-cheri/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Use latest Ubuntu as a base
+FROM ubuntu:20.04
+
```

Comment:
Reducing the number of RUN commands (and WORKDIR) reduces the size of the image. Maybe grouping them a little bit?

Reply:
Is the difference really that noticeable? I can make it all a one-liner, but then it will be harder to read.

Reply:
Like maybe 4-5 layers instead of 20 layers.